### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by Trust-007**
# Created a Gaming Conference site using the Cindy Shin design
- Added linters
- Used the MIT license
- Used the mobile first approach
- Used media queries to adapt the site for bigger screens
- Used javascript objects and the map function to create the speakers' section dynamically
- Created logic for mobile navigation

[Link to project description](https://www.loom.com/share/c18a312f077748fbbc1d947b632a8ce4)

### CODE REVIEW

File: html/about.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        </section>
+    </main>
+    <footer>
+        <div class="license">
+            @Gamers Oasis
+        </div>
+        <div class="rights">
+            2022 Gamers Oasis Nigeria. Some rights reserved.
+        </div>
+    </footer>
```
Comment (ha-manel) [inline html/about.html]: - [x] Great job on making the footer dark for the desktop version 👏 Kindly make the background light for mobile (only in the about page) as shown in the design guidelines 👍

## Your design:
image

## Design guidelines.
image

File: html/about.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>About Page</title>
```
Comment (ha-manel) [inline html/about.html:7]: - [x] Good job on not making the partners section for the about page in the desktop version 👏 Kindly add it for mobile view as shown in the design guidelines 👍 

## Your design:
image

## Design guidelines:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @Trust-007 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️
* Website looks great, I enjoyed the transitions and the animations ✨ 
* The code is clean and well organized 💯 

## Required Changes:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimalamiri — APPROVED** [review]
Hi @Trust-007, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

- The pages are designed very well ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

[Optional] In the original design the header background color is white, so please try to update your header background color in the mobile version.

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**